webpack/assets/javascripts/services/ChartService.consts.js

```diff
@@ -20,7 +20,7 @@ export const donutChartConfig = {
   tooltip: {
     show: true,
   },
-  legend: { show: false },
+  legend: { show: true },
```

Comment:
shouldn't we add legend on large charts only ?

Reply:
looks ok to me. how does it look to you?

Reply:
Its impact small size charts as well

In dashboard  you have legend information twice:
![donut_chart](https://user-images.githubusercontent.com/11807069/34765170-79907976-f5f9-11e7-91f6-dc4f630165b8.png)

IMHO adding legend into a small size chart is a bit redundant, because its too small. instead we could click to enlarge it, and see it.

![statistics](https://user-images.githubusercontent.com/11807069/34765085-3fe51790-f5f9-11e7-8d0d-4ba4fe874f97.png)